projects/igniteui-angular/src/lib/grids/summaries/summary-cell.component.ts

```diff
@@ -40,25 +44,20 @@ export class IgxSummaryCellComponent {
         return this.column.visibleIndex;
     }
 
-    @HostBinding('attr.tabindex')
-    public tabindex = 0;
-
     @HostBinding('attr.aria-describedby')
     public get describeby() {
         return `Summary_${this.column.field}`;
     }
 
-    @HostBinding('class.igx-grid-summary--active')
-    public focused: boolean;
-
-    @HostListener('focus')
-    public onFocus() {
-        this.focused = true;
+    @HostBinding('attr.id')
+    public get attrCellID() {
+        return `${this.grid.id}_${this.rowIndex}_${ this.visibleColumnIndex}`;
     }
 
-    @HostListener('blur')
-    public onBlur() {
-        this.focused = false;
+    @HostListener('pointerdown')
+    public activate() {
+        this.grid.navigation.activeNode = {row: this.rowIndex, column: this.visibleColumnIndex};
```

Comment:
Can we think of a way to use interface or class for the activeNode objects. This object assignment is used very often in the code